### PR TITLE
Add Wreck support for version 11 and above

### LINKED ISSUE #549: Wreck >=11.x.x currently unsupported
Wreck 11.x.x introduced a new format for the `response` event. It collapses the arguments into an object.

It was:
```
error, req, res, start, uri
```

It is now:
```
error, {req, res, start, uri}
```

See changelog history: https://github.com/hapijs/wreck/issues/162

A PR to support both formats is inbound.

### PR DESCRIPTION
**Pull request by lostthetrail**
Resolves #549

### CODE REVIEW

File: test/monitor.js [hunk 1]
```diff
@@ -1018,7 +1027,7 @@ describe('Monitor', () => {
 
                         expect(res.statusCode).to.equal(200);
                         // account for hapi15 differences
-                        expect(out.data.length).to.be.greaterThan(0);
+                        expect(out.data.length).to.be.greaterThan(1);
```
Comment (renegare) [inline test/monitor.js]: @lostthetrail I think this should be set to `least`. However; thats just to make it pass. Not entirely sure why one can't assert one event given you only make one wreck request in this single test case 🤔

Comment (lostthetrail) [inline test/monitor.js]: Drat. I had changed it from 0 to 1, when I was originally adjusting the unit test for wreck to confirm both cases (before I split it into two tests). I will revert that change in the next hour. Thanks!

File: lib/monitor.js [hunk 2]
```diff
@@ -68,6 +68,15 @@ class Monitor {
         if (this.settings.wreck) {
             this._wreckHandler = (error, request, response, start, uri) => {
 
+                // Wreck 11+ emits response event with signature (err, {req, res, start, uri})
+                const isWreckVersion11OrHigher = (!response && typeof request === 'object');
```
Comment (arb) [inline lib/monitor.js]: Thoughts on making this a proper `function` and checked `arguments.length` instead?

**Comment by arb** [comment]
CI failing.

**Comment by lostthetrail** [comment]
@arb Since this was a bug fix more than a new feature enhancement, I changed as little as possible.

Would you prefer if I adjusted from arrow to function and use the arguments list to dynamically determine values?

**Comment by arb** [comment]
Yeah I think so.

**Comment by lostthetrail** [comment]
You got it. Standby.

**Comment by lostthetrail** [comment]
@arb Back to you. Anything else you would like adjusted?

**Pull request merged**